docs/application/native/guides/machine-learning/machine-learning-inference.md

```diff
@@ -298,6 +298,135 @@ Followings are the available elements:
     char pipeline[] = "videotestsrc is-live=true ! videoconvert ! tensor_converter ! output-selector name=outs outs.src_0 ! tensor_sink name=sink0 async=false outs.src_1 ! tensor_sink name=sink1 async=false"
```

Comment:
File is already existing no need to update the Toc.
https://docs.tizen.org/application/native/guides/machine-learning/machine-learning-inference/